jetpack-start/wp-cli-api.php

```diff
@@ -63,9 +63,13 @@ public function connect( $args, $assoc_args ) {
 
 		$force_connection = WP_CLI\Utils\get_flag_value( $assoc_args, 'force', false );
 		if ( ! $force_connection && Jetpack::is_active() ) {
-			WP_CLI::error( 'Jetpack is already active; bailing. Run this command with `--force` to bypass this check or disconnect Jetpack before continuing.' );
+			if ( empty( Jetpack_Options::get_option( 'master_user' ) ) ) {
```

Comment:
Minor coding style thing: It's better to keep function calls (`Jetpack_Options::get_option( 'master_user' )`) out of conditionals for readability. Best to assign to a variable and then evaluate against that.

As a bonus, in the `else` case, this helps avoid an extra `get_option` call.